Add HTML and CSS Linter

Benmuiruri opened the pull request:
Added HTML and CSS Linters. This commit is to test whether the Linter is correctly installed.

Benmuiruri commented:
To whom it may concern, 

Hello Code Reviewer :smiley: quick heads up that I have put max-width on the hero H1 because I was building for mobile. When I update the website for both mobile and Web view, I will put the max-width in a media tag. 

Thanks :+1:

Donard97 reviewed (changes requested):
## Hello @Benmuiruri , ✋  

You did an amazing job so far! 💯 💯 

There are some issues that you still need to work on to go to the next project but you are almost there! 👍 

## Highlights

- README is proffesional ✔️ 
- Gitflow is followed ✔️ 

![giphy (1)](https://user-images.githubusercontent.com/74506933/153423219-41f7b2c6-95d4-48ee-b289-7fed4ab121f6.gif)

## Required Changes ♻️

- [ ] Kindly change PR and do it more descriptive and let us know in details what you did in this branch
- [ ] Please add linters files like .stylelintrc and .hintrc following [linter configuration repository](https://github.com/microverseinc/linters-config/tree/master/html-css)

**_Check the comments under the review._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Donard97 commented:
> To whom it may concern,
> 
> Hello Code Reviewer 😃 quick heads up that I have put max-width on the hero H1 because I was building for mobile. When I update the website for both mobile and Web view, I will put the max-width in a media tag.
> 
> Thanks 👍

@Benmuiruri   Thank you for letting me know. You don't need right now that max-width that you put. Even without that, it is right. Keep rocking! 💪

Benmuiruri commented:
Hi @Donard97 thanks for the review. Please help me understand some of the requested changes. 

1. How do I change PR and do it more descriptively? Do you mean changing the title from "Add HTML and CSS Linter #1" to something more descriptive? 
2. I don't understand what you mean by "Please add linters files likestylelintrc and .hintrc following [linter configuration repository](https://github.com/microverseinc/linters-config/tree/master/html-css). I followed the steps on that webpage to add the linters. Please clarify what you mean, I am still learning :pray: 

As I wait for those clarifications, let me fix the other two code-related issues. 

Thanks.

Donard97 commented:
> Hi @Donard97 thanks for the review. Please help me understand some of the requested changes.
> 
> 1. How do I change PR and do it more descriptively? Do you mean changing the title from "Add HTML and CSS Linter [Add HTML and CSS Linter #1](https://github.com/Benmuiruri/my-portfolio/pull/1)" to something more descriptive?
> 2. I don't understand what you mean by "Please add linters files likestylelintrc and .hintrc following [linter configuration repository](https://github.com/microverseinc/linters-config/tree/master/html-css). I followed the steps on that webpage to add the linters. Please clarify what you mean, I am still learning 🙏
> 
> As I wait for those clarifications, let me fix the other two code-related issues.
> 
> Thanks.

Hi @Benmuiruri, of course. I texted you on slack.

Benmuiruri commented:
Hi @Donard97 I made the requested changes 

- updated background image position to center 
- took the logo out of the `.flex-nav ul`
- Added the linters files. 

Thanks

Benmuiruri commented:
Thanks, @ReshyResh, it feels great to have completed the first part of the project.
